Translation/Extractor/File/FormExtractor.php

```diff
@@ -101,12 +101,23 @@ public function enterNode(\PHPParser_Node $node)
                     }
                     continue;
                 }
+                
+                if ('placeholder' === $item->key->value && $item->value instanceof \PHPParser_Node_Expr_ConstFetch
+                    && $item->value->name instanceof \PHPParser_Node_Name && 'false' === $item->value->name->parts[0]) {
+                    continue;
+                }
+                if ('placeholder' === $item->key->value && $item->value instanceof \PHPParser_Node_Expr_Array) {
+                    foreach ($item->value->items as $sitem) {
+                        $this->parseItem($sitem, $domain);
+                    }
+                    continue;
+                }
 
                 if ('choices' === $item->key->value && !$item->value instanceof \PHPParser_Node_Expr_Array) {
                     continue;
                 }
 
-                if ('label' !== $item->key->value && 'empty_value' !== $item->key->value && 'choices' !== $item->key->value && 'invalid_message' !== $item->key->value && 'attr' !== $item->key->value ) {
+                if ('label' !== $item->key->value && 'empty_value' !== $item->key->value && 'placeholder' !== $item->key->value && 'choices' !== $item->key->value && 'invalid_message' !== $item->key->value && 'attr' !== $item->key->value ) {
                     continue;
                 }
```

Comment:
This is the line causing the failures. I've spent about 2-3 hours now investigating this. Oddly enough the only thing I know is that this is the line causing the problem. I have another set of changes to clean this whole area of code up. However I'm wondering why this change on this line was added by you. The tests pass without it so I'm wondering if you don't apply that, do you get the extracted data you wanted?